hstream-store/HStream/Store/Internal/LogDevice/LogConfigTypes.hs

```diff
@@ -752,7 +753,7 @@ foreign import ccall unsafe "hs_logdevice.h ld_loggroup_update_extra_attrs"
     -> Ptr LogDeviceLogGroup
     -> Int -> BAArray# Word8 -> BAArray# Word8
     -> StablePtr PrimMVar -> Int -> Ptr LogsConfigStatusCbData
-    -> IO ErrorCode
+    -> IO CInt
```

Comment:
Why change this to `CInt`?

Reply:
The return value indicates whether the async request has been successfully scheduled. (if I understood it correctly)
https://github.com/hstreamdb/LogDevice/blob/13679a020679b63c0c2af90a96fb193a7afbf92a/logdevice/include/Client.h#L1248